langchain4j-weaviate/src/main/java/dev/langchain4j/store/embedding/weaviate/WeaviateEmbeddingStore.java

```diff
@@ -81,6 +82,7 @@ public WeaviateEmbeddingStore(
             String objectClass,
             Boolean avoidDups,
             String consistencyLevel,
+            String metadataFieldName,
```

Comment:
Could you please move this field to the end of parameters, right after `textFieldName`?
(The same for javadoc above)

Reply:
Done